### PR TITLE
[GraFx Studio] Number variables

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

File: docs/GraFx-Studio/guides/template-variables/define/index.md [hunk 1]
```diff
@@ -59,14 +59,32 @@ This location does not have any effect on the working of your document. It will
...
 
+Create a variable. Then, in the properties panel, choose the variable type. Different use cases demand different variable types.
+
+![screenshot](vartypes.png)
+
 ### Single Line of Text
 
 A placeholder for text.
 
+[Use Text variables](/GraFx-Studio/guides/template-variables/define/)
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/define/index.md:70]: This link refers to the same page.

File: docs/GraFx-Studio/guides/template-variables/define/index.md [hunk 2]
```diff
@@ -59,14 +59,32 @@ This location does not have any effect on the working of your document. It will
 
 ## Types
 
+Create a variable. Then, in the properties panel, choose the variable type. Different use cases demand different variable types.
+
+![screenshot](vartypes.png)
+
 ### Single Line of Text
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/define/index.md:66]: Should be "Single line text"

File: docs/GraFx-Studio/guides/template-variables/number/index.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+#### Decimal Separator
+
+The symbol used to separate the integer part and the decimal part.
+
+#### Thousands Separator
+
+The “thousands separator” symbol is used to group digits into sets of three for easier readability of large numbers, commonly represented by a comma (,), period (.), or other symbols depending on regional conventions.
+
+## Set Style Properties
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/number/index.md:63]: We call them "Appearance settings"

File: docs/GraFx-Studio/guides/template-variables/number/index.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Set the number of decimal places that will be displayed.
+
+#### Decimal Character Style
+
+![screenshot-full](ds0.png)
+
+You can define a character style that is different from the style defined for the text, to use for the decimal part of the number.
+
+The default Paragraph style for the frame is "Text".
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/number/index.md:43]: I would mention that it is an example. Now I first read it as if there is always a default paragraph style, which isn't the case.

File: docs/GraFx-Studio/guides/template-variables/number/index.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+![screenshot-full](ds1.png)
+
+A specific character style is defined and has the property to show as "Superscript".
+
+![screenshot-full](ds2.png)
+
+The variable decimal setting shows the decimals will use this character setting for the decimal part.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/number/index.md:51]: Also important to know is the style inheritance model:
- If you want to use the decimal character style, you need to make sure that there are no style overrides on the number variable (you can use the "Clear style overrides" button in the text properties panel to remove the style overrides)
- If a character style is already applied, it is locally replaced by the decimal character style

**Review by jeroencosyn — CHANGES_REQUESTED** [review]

File: docs/GraFx-Studio/guides/template-variables/define/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/define/index.md:70]: Will remove the link (for now), page does not exist yet. Will re-add later

File: docs/GraFx-Studio/guides/template-variables/define/index.md (hunk 2 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/define/index.md:66]: Done

File: docs/GraFx-Studio/guides/template-variables/number/index.md (hunk 3 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/number/index.md:63]: CHeck

File: docs/GraFx-Studio/guides/template-variables/number/index.md (hunk 4 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/number/index.md:43]: Made it more clear, and used it as example

File: docs/GraFx-Studio/guides/template-variables/number/index.md (hunk 5 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/number/index.md:51]: check, added clarification

File: docs/GraFx-Studio/guides/template-variables/number/index.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+The chosen Paragraph style for the frame is "Text".
+
+![screenshot-full](ds1.png)
+
+A specific character style is defined and has the property to show as "Superscript".
+
+![screenshot-full](ds2.png)
+
+The variable decimal setting shows the decimals will use this character setting for the decimal part.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/guides/template-variables/number/index.md:53]: This sentence is a bit confusing. I would write it as something like: The decimal character style is set to "Superscript" to use it as character style for the decimal part.

**Review by jeroencosyn — APPROVED** [review]

File: docs/GraFx-Studio/guides/template-variables/number/index.md (hunk 6 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/template-variables/number/index.md:53]: Changed it

**Pull request merged**